.github/workflows/lint-markdown.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - "**.md"
+      - "**.mdx"
+  push:
+    branches:
+      - main
+    paths:
+      - "**.md"
+      - "**.mdx"
+
+jobs:
+  md-lint:
```

Comment:
convention for jobs name is `[workflow name]-[jobs name]` this should be `lint-md`.

Would appreciate if you can note this down so I don't have to nit/fix it everytime